index.html

```diff
@@ -30,6 +30,25 @@
 -->
 
 <div class="container">
+  <div class="row">
+    <div class="col-md-12">
+      <div class="col-md-1">
+      </div>
+      <div class="col-md-10 lead text-justify">
+        Interledger is an open protocol suite for sending payments across different <em>ledgers</em>.
```

Comment:
I find it easier to understand the concept if we use the phrase "payment networks" instead of ledgers here although I get that you're also trying to introduce the core concepts, of which a ledger is one.